go-controller/pkg/ovn/master.go

```diff
@@ -889,6 +890,9 @@ func (oc *DefaultNetworkController) addUpdateRemoteNodeEvent(node *kapi.Node, sy
 			}
 		}
 	}
+	if err != nil {
+		klog.Infof("remote node added/updated: %s", node.Name)
```

Comment:
hmm why do we have this log line for error not != nil? won't we return that error which will get percolated upwards?
because at this stage we haven't really added or updated the node right? -> either zoneICHandler.AddRemoteZoneNode failed or zoneChassisHandler.AddRemoteZoneNode failed or something?
also if we want to keep this for debugging purposes should we add the error to the log?

Reply:
Yes, this looks weird.
I would have at the beginning of the function an unconditional log (I believe they have to start with capital letter)
```
klog.Infof("Remote node added/updated: %s", node.Name)
```
And then if it was a local node
```
if present {
    klog.Infof("Node moved out of the local zone: %s", node.Name)
```
Something like that.

Reply:
this was something i was using for debugging. I'll remove it and add a log somewhere else.